src/plugins/GUITestBase/src/GTUtilsTaskTreeView.cpp

```diff
@@ -236,11 +252,31 @@ int GTUtilsTaskTreeView::countTasks(HI::GUITestOpStatus& os, const QString& item
 }
 #undef GT_METHOD_NAME
 
+#define GT_METHOD_NAME "getTaskStatus"
 QString GTUtilsTaskTreeView::getTaskStatus(GUITestOpStatus& os, const QString& itemName) {
     openView(os);
     GTGlobals::sleep(500);
     return getTreeWidgetItem(os, itemName)->text(1);
 }
+#undef GT_METHOD_NAME
+
+#define GT_METHOD_NAME "getTaskProgress"
+int GTUtilsTaskTreeView::getTaskProgress(HI::GUITestOpStatus& os, const QString& itemName, bool failOnNull) {
+    if (!isViewOpened(os)) {
+        openView(os);
+        GTGlobals::sleep(500);
+    }
+    auto item = getTreeWidgetItem(os, itemName, failOnNull);
+    CHECK(item != nullptr, -1);
+
+    auto itemText = item->text(2);
+    bool ok = false;
+    auto number = itemText.left(2).toInt(&ok);
```

Comment:
What if the progress is `1%`? Will `left(2)` work in this case?

Reply:
OK